quizapp/src/main/java/billennium/quizapp/ActiveProfileVerifier.java

```diff
@@ -0,0 +1,41 @@
+package billennium.quizapp;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.context.event.ApplicationEnvironmentPreparedEvent;
+import org.springframework.context.ApplicationListener;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static org.apache.commons.lang3.StringUtils.join;
+
+public class ActiveProfileVerifier implements ApplicationListener<ApplicationEnvironmentPreparedEvent> {
+
+    private static final Logger LOG = LoggerFactory.getLogger(ActiveProfileVerifier.class);
+
+    private List<String> availableProfiles = Arrays.asList("dev", "test");
+
+    @Override
+    public void onApplicationEvent(ApplicationEnvironmentPreparedEvent event) {
+        String[] activeProfiles = event.getEnvironment().getActiveProfiles();
+        if (noProfiles(activeProfiles) || notAcceptedProfile(activeProfiles)) {
+            LOG.error("Spring application active profile is not correct! " +
+                            "provided [{}], available [{}]",
+                    join(activeProfiles, ","),
```

Comment:
to samo można zrobić Collectors.joining() wydaje mi się i wtedy cala zależność `apache.commons` wypada

Reply:
Dobry pomysł

Reply:
znalazłem jeszcze inną metodę: String join